### DISCUSSION TITLE
Display Subfeature ID on GFF3 Track

### DISCUSSION THREAD

**Discussion by singhsum-nih** [GMOD/jbrowse-components#4306] [Q&A] [answered]
Is it possible to display the subfeature ID (transcript ID) in addition to gene ID on GFF3 tracks?  I see they are available in the feature details panel but wanted to check if there was a way, out of the box, to display them on the tracks as well.  Thanks!

**Comment by cmdcolin** [comment]
@singhsum-nih hi there
currently this is not implemented yet but we definitely want to add this

https://github.com/GMOD/jbrowse-components/issues/241

**Reply by cmdcolin** [reply, marked as answer]
if you want to subscribe to that issue you can get updates!

**Reply by singhsum-nih** [reply]
Thank you so much!  I will do that.
